platforms/Windows/cli/cli.wxs

```diff
@@ -222,9 +216,6 @@
       <Component>
         <File Source="$(TOOLCHAIN_ROOT)\usr\bin\PackageModel.dll" />
       </Component>
-      <Component>
-        <File Source="$(TOOLCHAIN_ROOT)\usr\bin\PackageModelSyntax.dll" />
```

Comment:
It looks like `PackageModelSyntax` is still able to go either way in https://github.com/swiftlang/swift-package-manager/pull/8268/files and https://github.com/swiftlang/swift-package-manager/blob/2215d098ed2aa991ed3f3d8c712266f14ebe4b45/Sources/PackageModelSyntax/CMakeLists.txt#L9

Does that PR need to switch it to a static archive, or are you switching things to use `-DBUILD_SHARED_LIBS=NO`?

Reply:
Nice catch! That should still be dynamic - I believe it ended up being that this was not profitable.

Reply:
Okay, now it does.